spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -47,25 +47,28 @@
 
   scenario "User deletes a step by step page", js: true do
     given_there_is_a_step_by_step_page_with_steps
-    and_I_visit_the_index_page
```

Comment:
Don't you need this step to visit the step-by-step overview page?

Should the test read:
```
and_I_visit_the_index_page
and_I_visit_the_publish_or_delete_page
```

So that it mirrors the action that the user takes?